pkgs/by-name/ca/cabal-cli/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+}:
+
+let
+  version = "15.0.2";
+  pname = "cabal-cli";
+in
+buildNpmPackage {
+  inherit version pname;
+
+  src = fetchFromGitHub {
+    owner = "cabal-club";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-DZ+F+pgPc8WuiBhLgxNO5es5lA996fJdvZyg6QjfrHg=";
+  };
+
+  npmDepsHash = "sha256-Oqyx6pytDrYg1JbHawKxnnWEJxaFUaM9LcREizh3LFQ=";
+
+  dontNpmBuild = true;
+
+  NODE_OPTIONS = "--openssl-legacy-provider";
```

Comment:
Prefix environment variables with `env.`, that's preferred.

Reply:
Thanks, I have fixed it.